### PR TITLE
Services hero

### PR DESCRIPTION
**Pull request by SheCode287**
Updated folder Hero and Services and their file paths in Home folder

### CODE REVIEW

File: src/components/hero/Hero.jsx
Comment (Ozyugoo) [inline src/components/hero/Hero.jsx:1]: Kindly check, which of the hero files are you sending? Please send the one you corrected and delete the other one from our repo. Once it is done, we can merge them.

Comment (SheCode287) [reply src/components/hero/Hero.jsx:1]: I did exactly, that.

Comment (SheCode287) [reply src/components/hero/Hero.jsx:1]: And i don't have permission to delete files from dev, so kindly as soon as you see this message remove the files that are not in any folder i.e services and hero

**Review by Ozyugoo — APPROVED** [review]
It seems okay now, lets look at it

**Pull request merged**